### PR TITLE
The first draft of the Bulk Generator section #33

### PR DESCRIPTION
**Pull request by irinaponomarevahaulmont**
Please review the first draft.

### CODE REVIEW

File: content/modules/studio/pages/creating-graphql-controllers.adoc [hunk 1]
```diff
@@ -133,4 +133,67 @@ Once we've set all the parameters, we click *_OK_* and the controller will be cr
...
+[[capabilities]]
+You can also check or uncheck the CRUD operations depending on which ones you actually need, as well as the load list capabilities. For example, if you don't need Filtering and Sorting to be supported on your controllers, and only intend to use Pagination, you set the checkboxes as follows:
+
+image::graphql/load-list-capabilities.png[align=center, width=600]
+
+In this case, the resulting souce code of the controller will only contain the methods needed for Pagination.
+
+Once you have configured everything as necessary, click *_OK_*. The controllers for the chosen repositories will be created.
+
+For simplicity, this dialog window contains a reduced number of parameters in comparison with the dialog used to create one controller at a time. For example, it offers filtering by local attributes only, but not by associations. So, if you need to set up your CRUD controller more in depth, you need to use the ordinary dialog available under *_Spring for GraphQL_* -> *_GraphQL Controllers by Repository_* and create the controllers one by one.
```
Comment (alexbudarov) [inline content/modules/studio/pages/creating-graphql-controllers.adoc:166]: GraphQL Controllers by Repository
-->
GraphQL Controller by Repository

File: content/modules/studio/pages/creating-graphql-controllers.adoc [hunk 2]
```diff
@@ -133,4 +133,67 @@ Once we've set all the parameters, we click *_OK_* and the controller will be cr
...
+
+The load list capabilities group allows us to configure the functionality offered by the list load operation, as we have already xref:#capabilities[mentioned above]:
+
+ * Filtering by local attributes
+ * Pagination
+ * Sorting by local attributes
+
+image::graphql/capabilities.png[align=center, width=600]
+
+If you need more detailed configuration that specifies which attributes can be used for sorting and filtering or makes use of pagination options, use the dialog available under *_Spring for GraphQL_* -> *_GraphQL Controllers by Repository_* to create controllers individually.
```
Comment (alexbudarov) [inline content/modules/studio/pages/creating-graphql-controllers.adoc:197]: GraphQL Controllers by Repository
-->
GraphQL Controller by Repository

File: content/modules/studio/pages/creating-graphql-controllers.adoc [hunk 3]
```diff
@@ -133,4 +133,67 @@ Once we've set all the parameters, we click *_OK_* and the controller will be cr
...
+* Load list of items
+* Load item by id
+* Save item
+* Delete item
+
+Using the checkbox located next to each operation we choose the operations, which will be implemented within the controller.
+
+image::graphql/crud-operations.png[align=center, width=600]
+
+The load list capabilities group allows us to configure the functionality offered by the list load operation, as we have already xref:#capabilities[mentioned above]:
```
Comment (alexbudarov) [inline content/modules/studio/pages/creating-graphql-controllers.adoc:189]: > load list capabilities

Style as caption text

**Review by alexbudarov — CHANGES_REQUESTED** [review]

File: content/modules/studio/pages/creating-graphql-controllers.adoc [hunk 4]
```diff
@@ -133,4 +133,67 @@ Once we've set all the parameters, we click *_OK_* and the controller will be cr
...
+
+image::graphql/load-list-capabilities.png[align=center, width=600]
+
+In this case, the resulting souce code of the controller will only contain the methods needed for Pagination.
+
+Once you have configured everything as necessary, click *_OK_*. The controllers for the chosen repositories will be created.
+
+For simplicity, this dialog window contains a reduced number of parameters in comparison with the dialog used to create one controller at a time. For example, it offers filtering by local attributes only, but not by associations. So, if you need to set up your CRUD controller more in depth, you need to use the ordinary dialog available under *_Spring for GraphQL_* -> *_GraphQL Controllers by Repository_* and create the controllers one by one.
+
+The repository table displays for you the list of all repositories, for which there are no controllers created.
```
Comment (alexbudarov) [inline content/modules/studio/pages/creating-graphql-controllers.adoc:168]: > all repositories

all Spring Data repositories

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Comment by alexbudarov** [comment]
You need to restructure the content a little bit. Not just record what I say in a call. This is your role - to think how to structure the content so it doesn't contain duplications, contains information in a best form to be understood by the reader.

For example, similar information about the same thing - "Load list capabilities" is written twice:

1) 

> You can also check or uncheck ... , as well as the load list capabilities. For example, if you don’t need Filtering and Sorting to be supported on your controllers, and only intend to use Pagination, you set the checkboxes as follows:
 << screenshot >>
In this case, the resulting souce code of the controller will only contain the methods needed for Pagination.

2) Second time - below.

> The load list capabilities group allows us to configure the functionality offered by the list load operation, as we have already mentioned above:
  Filtering by local attributes
  Pagination
  Sorting by local attributes
 << very similar screenshot >>
 If you need more detailed configuration that specifies which attributes can be used for sorting and filtering or makes use of pagination options, use the dialog available under Spring for GraphQL → GraphQL Controllers by Repository to create controllers individually. 

The same about table of repositories - they are described twice, a bit differently.
What happens after clicking OK - also described twice.

**Comment by irinaponomarevahaulmont** [comment]
Please review the changes.

File: content/modules/studio/pages/creating-graphql-controllers.adoc [hunk 5]
```diff
@@ -133,4 +133,65 @@ Once we've set all the parameters, we click *_OK_* and the controller will be cr
...
+* Load list of items
+* Load item by id
+* Save item
+* Delete item
+
+Using the checkbox located next to each operation we choose the operations, which will be implemented within the controller.
+
+image::graphql/crud-operations.png[align=center, width=600]
+
+The *_load list capabilities_* group allows us to configure the functionality offered by the list load operation:
```
Comment (alexbudarov) [inline content/modules/studio/pages/creating-graphql-controllers.adoc:179]: load  --> Load

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Pull request merged**